tempodb/encoding/vparquet4/block_autocomplete.go

```diff
@@ -922,15 +922,20 @@ func (d *distinctAttrCollector) KeepGroup(result *parquetquery.IteratorResult) b
 	var val traceql.Static
 
 	for _, e := range result.Entries {
+		if len(e.Values) == 0 {
```

Comment:
Autocomplete suggests the elements of the array as single values. Since we don't have array literals in TraceQL this is probably the most useful behavior